### ISSUE TITLE
There are a few audit task interactions with empty pano ids

### ISSUE THREAD

**Issue by misaugstad** [ProjectSidewalk/SidewalkWebpage#1190]
@johannkm brought this to my attention, that audit task interactions' pano id are empty strings for a small number of entries. This looks to be only about 19,000 entries (< 0.01% of interactions), so it doesn't seem to be a big deal at all. Just wanted to log it here.

**Comment by maddalihanumateja** [comment]
How recent are those audit tasks? This is from the turker vs volunteer HITs right?

**Comment by misaugstad** [comment]
About half of them come from anonymous users on 5/16/17. Those are the most recent ones, and the rest are mostly from anon users, quite a few are from @manaswis

**Comment by misaugstad** [comment]
Taking a look at this again, it looks like we don't have pano ids for 0.05% of interactions. And this is still occurring. Would be worth investigating.

**Comment by andrew4699** [comment]
We default the panoId to null when we fail to retrieve it from the map object and I got `Cannot read property 'getPanoId' of undefined` when logging the cause of failure for the line `svl.map.getPanoId()`. This was after I reloaded the page.

A lot of this is probably due to the map not being loaded at the time of logging because a lot of the null IDs are for events that happen near page load/unload like TaskStart/Unload/RefreshTracker. I'll continue investigating the times it happened for other events.
